src/services/school/model.js

```diff
@@ -24,6 +24,20 @@ const schoolSchema = new Schema({
 	timestamps: true
 });
 
+const yearSchema = new Schema({
```

Comment:
Can you explain, why we should put this into the School-Model file?  `year` and especially `grade` sound like userGroups to me ... In the past we considered to create a grade-model next to the other userGroup-models, but there was no application this time.

Reply:
The goal of the year schema is to provide a reference to, well, school years.
(for example "this course is held in the year '2017/18').
In the long run, we would like schools to be able to adjust the definition of those years to their needs: they might use the 'normal' school year circle, they might use half years, quarters, semesters, the year might start or end at different times depending on the federal state etc.

Thats why I put them with the school, because they might be defined on a school-by-school basis. (event though the code for that doesnt exist, as we figured its not a priority right now).

For grade levels its a similar thought, some have 10, some have 12, some 13, and maybe we will see a completely different model one day.